lesson13/lesson13_script2.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# detects how many files were created in the directory
+ls | wc -l
+
+# replaces all permissions set to 777 with 664
+for file in /home/dmitryyankov/lesson13/*
+do
+	if [ $(stat -c "%a" "$file") == "777" ]
+	then
+		chmod 664 $file
+	fi
+done
+
+# detects empty files and removes them
+find /home/dmitryyankov/lesson13 -type f -size 0 -print -delete
+
+# removes all lines except the first one
+for file in /home/dmitryyankov/lesson13/*
```

Comment:
script should create folder and files should be located inside. currently, script will work against all files in lesson13 directory (it will also remove lines from script itself)

Reply:
done